### ISSUE TITLE
Broken images in HTML report

### ISSUE THREAD

**Issue by spitschan** [poldracklab/fmriprep#1121]
I ran the following anatomy-only command with with `fmriprep-docker` (FMRIPREP 1.0.11):

```
 fmriprep-docker --fs-license-file fslicense.txt --anat-only /Users/spitschan/Projects/hypImaging/data/bids /Users/spitschan/Projects/hypImaging/data/bids/out/
```

The report and the underlying SVG images all show an inverted question mark:

![screen shot 2018-05-10 at 10 34 19](https://user-images.githubusercontent.com/3341551/39863157-c8816b6a-543d-11e8-9f82-0c47286f1327.png)

Any suggestions?

**Comment by oesteban** [comment]
I think that was fixed in the latest 1.0.12. Could you give it a go?

**Comment by spitschan** [comment]
That fixed it. Thanks @oesteban.
